src/main/java/org/stellar/sdk/Memo.java

```diff
@@ -24,10 +26,18 @@ public static MemoNone none() {
 
     /**
      * Creates new {@link MemoText} instance.
-     * @param text
+     * @param text test in utf8 encoding
      */
     public static MemoText text(String text) {
-        return new MemoText(text);
+        return new MemoText(text, Charset.forName("UTF8"));
+    }
+
+    /**
+     * Creates new {@link MemoText} instance.
+     * @param text test in unknown encoding
```

Comment:
should this be "text in unknown encoding" instead of "test in unknown encoding?"